### PR TITLE
GLTFExporter: Support for binary (.glb) export option.

### PR DESCRIPTION
**Pull request by donmccurdy**
/cc @fernandojsg

### CODE REVIEW

File: examples/misc_exporter_gltf.html [hunk 1]
```diff
@@ -48,13 +49,22 @@
...
-					saveString( output, 'scene.gltf' );
+					if ( result instanceof ArrayBuffer ) {
+
+						saveArrayBuffer( result, 'scene.glb' );
+
+					} else {
+
+						var output = JSON.stringify( result, null, 2 );
+						console.log( output );
+						saveString( output, 'scene.gltf' );
```
Comment (fernandojsg) [inline examples/misc_exporter_gltf.html]: Maybe modifying the extension to `glb` if binary is selected?

**Review by fernandojsg — COMMENTED** [review]
Looking nice 👍 I'm just not sure if the glb format support external references to images or they need to be added inside the binary /cc @donmccurdy

**Comment by fernandojsg** [comment]
r+

File: examples/misc_exporter_gltf.html (hunk 1 above)
Comment (donmccurdy) [reply examples/misc_exporter_gltf.html]: ^Handled above this section

**Comment by donmccurdy** [comment]
> I'm just not sure if the glb format support external references to images or they need to be added inside the binary

It's allowed, you can have any combination of glb or gltf with or without embedded images. But  once we have inline images supported in GLTFExporter we should make sure they're in the binary payload and not Data URIs yes

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**